File: Python/lista1 - estrutura de decisao/exercicio_8.py
grade_1 = float(input('Informe a nota 1: '))
grade_2 = float(input('Informe a nota 2: '))

average = (grade_1 + grade_2) / 2

if average >= 7:
    print('O aluno foi aprovado.')
else:
    print('O aluno está de exame.')

File: C/lista1 - estrutura de decisao/exercicio_8.c
#include <conio.h>
#include <stdio.h>

int main() {
	float grade1, grade2, average;
	
	printf("Informe a nota 1: ");
	scanf("%f", &grade1);
	printf("Informe a nota 2: ");
	scanf("%f", &grade2);
	
	average = (grade1 + grade2) / 2;
	
	if (avrage >= 7) {
		printf("O aluno esta aprovado(a).");
	} else {
		printf("O aluno(a) esta de exame.");
	}
	
	getch();
	return 0;
}